rome-modules/src/main/java/com/rometools/modules/atom/io/AtomModuleGenerator.java

```diff
@@ -52,46 +54,45 @@ public final String getNamespaceUri() {
     public void generate(Module module, Element element) {
         if (module instanceof AtomLinkModule) {
             AtomLinkModule m = (AtomLinkModule) module;
-            generateLinc(m.getLink(), element);
+            generateLinks(m.getLinks(), element);
         }
     }
 
-    private void generateLinc(Link link, Element element) {
-        if (link == null) {
-            return;
-        }
-
-        Element linkElement = new Element("link", NS);
-
-        if (link.getHref() != null) {
-            Attribute href = new Attribute(AtomLinkAttribute.HREF, link.getHref());
-            linkElement.setAttribute(href);
-        }
-        if (link.getType() != null) {
-            Attribute type = new Attribute(AtomLinkAttribute.TYPE, link.getType());
-            linkElement.setAttribute(type);
-        }
-        if (link.getRel() != null) {
-            Attribute rel = new Attribute(AtomLinkAttribute.REL, link.getRel());
-            linkElement.setAttribute(rel);
-        }
-
-        if (link.getHreflang() != null) {
-            final Attribute hreflangAttribute = new Attribute(AtomLinkAttribute.HREF_LANG, link.getHreflang());
-            linkElement.setAttribute(hreflangAttribute);
-        }
-
-        if (link.getTitle() != null) {
-            final Attribute title = new Attribute(AtomLinkAttribute.TITLE, link.getTitle());
-            linkElement.setAttribute(title);
-        }
-
-        if (link.getLength() != 0) {
-            final Attribute lenght = new Attribute(AtomLinkAttribute.LENGTH, Long.toString(link.getLength()));
-            linkElement.setAttribute(lenght);
+    private void generateLinks(List<Link> links, Element element) {
+        for (Link link : links) {
```

Comment:
Maybe extract `generateLink` method? Then it will be easier to see what `generateLinks` does.